scripts/github_runner/setup_bare_metal_runner.sh

```diff
@@ -36,6 +36,16 @@ GH_ORG=$1
 GH_PAT=$2
 SANDBOX=$3
 
+# clean the service file before proceeding
+pattern="/etc/systemd/system/actions.runner.vhive-serverless-vhive.*.service"
+if ls $pattern 1> /dev/null 2>&1; then
+  rm $pattern
+  echo "Deleted matching files."
```

Comment:
be specific which files, i.e., belong to what entity

Reply:
Done